### ISSUE TITLE
SELECT without INTO, false positive

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1924]
```abap
    SELECT COUNT(*)
      FROM tcdrp
      WHERE object = mv_object.
```

https://rules.abaplint.org/obsolete_statement/
